src/prefect/settings.py

```diff
@@ -641,6 +641,32 @@ def default_cloud_ui_url(settings, value):
 task will refresh the cached results. Defaults to `False`.
 """
 
+PREFECT_TASK_DEFAULT_RETRIES = Setting(int, default=0)
+"""
+This value sets the retries for all tasks associated with a flow. 
+This value does not overwrite individually set retries values on tasks
+"""
+
+PREFECT_FLOW_DEFAULT_RETRIES = Setting(int, default=0)
+"""
+This value sets the retries for all flows. 
+This value does not overwrite individually set retries values on a flow
+"""
+
+PREFECT_FLOW_DEFAULT_RETRY_DELAY_SECONDS = Setting(Union[int, float], default=0)
+"""
+This value sets the retry delay seconds for all flows when this environment is set
```

Comment:
```suggestion
This value sets the retry delay seconds for all flows.
```

Reply:
Really could just be "The default retry delay seconds for all flows." — "This value sets" is pretty redundant.